homeassistant/components/glances/sensor.py

```diff
@@ -344,6 +354,24 @@ def available(self) -> bool:
     @property
     def native_value(self) -> StateType:
         """Return the state of the resources."""
+        if self.entity_description.type == "uptime":
+            uptime = self.coordinator.data[self.entity_description.type]
```

Comment:
I would suggest isolating this code in a separate function as it is specfific to the uptime sensor